conf/binac_smp.config

```diff
@@ -0,0 +1,23 @@
+//Profile config names for nf-core/configs
```

Comment:
I think cluster configs should be in [https://github.com/nf-core/configs](https://github.com/nf-core/configs)

Reply:
Absolutely!

However, to assemble a complex metagenome SPAdes needed ~350 GB memory (and MEGAHIT <40 GB), which isnt provided by the cluster queue in the current nf-core/configs/binac.config. Therefore I had to do an additional config that specifies for that cluster a queue for high memory jobs and integrated it (temporary) here. Ultimately I would like to use the multiple queue logic discussed here: https://github.com/nf-core/configs/issues/39
I honestly dont know how we could right now make the workflow work for the cluster I work with, without that specific config. And I certainly dont want to have that in nf-core/configs since it is only necessary for this pipeline and might be not needed in future at all.